github/repos.go

```diff
@@ -75,6 +75,7 @@ type Repository struct {
 	AssigneesURL     *string `json:"assignees_url,omitempty"`
 	BlobsURL         *string `json:"blobs_url,omitempty"`
 	BranchesURL      *string `json:"branches_url,omitempty"`
+	CodeOfConduct    *string `json:"code_of_conduct,omitempty"`
```

Comment:
Also, don't put it in the `// API URLs` section since it's not an API URL. Put it at the top with all the main fields... Maybe right under `Homepage` field?